webapp/crashstats/crashstats/tests/test_views.py

```diff
@@ -438,15 +458,13 @@ def mocked_processed_crash_get(**params):
         assert "OS X 10.11" in content
 
         # the protected data will appear if we log in
-        user = self._login()
-        group = self._create_group_with_permission("view_pii")
-        user.groups.add(group)
-        assert user.has_perm("crashstats.view_pii")
+        user = user_helper.create_protected_user()
```

Comment:
The `user_helper` makes creating users with appropriate permissions much easier.